apps/bbsync/query.py

```diff
@@ -237,10 +238,8 @@ def generate_hightouch_flags(self):
             Flaw.FlawMajorIncident.CISA_APPROVED: ("-", "+"),
             # flags NOVALUE and INVALID are ignored
         }
-
         if self.flaw.major_incident_state in flags_to_write:
             hightouch, hightouch_lite = flags_to_write[self.flaw.major_incident_state]
-
```

Comment:
You probably deleted these lines by mistake.

Reply:
Yes. I thought it was `black` but it was me.